### DISCUSSION TITLE
replace const with 5

### DISCUSSION THREAD

**Discussion by mh-firouzjah** [TodePond/GulfOfMexico#410] [Q&A]
what if I say:

    // const const 5 = 4!
    // print(2 + 2 === 5)! //true
    const const 5 = const! // 😂🤦🏻‍♂

**Comment by TodePond** [comment]
heya this would be implicitly interpreted as a string

```
const const 5 = "const"!
```

**Reply by ShreyanJain9** [reply]
DreamBerd should let you do `const const` implicitly, and then instead of using `==` for comparison, just use `=` bcz immutability!

**Reply by circle-gon** [reply]
Can keywords not be renamed in DreamBerd? I thought the "other language constructs" part of the Naming section implied that you _could_ rename keywords.

**Comment by Forest-Terminal** [comment]
I think the code should look like this:
```
//  5 = 4!  <- 5 is already set to 5 so no declaration is needed
// print(2 + 2 === 5)! //true
5 = const! // 😂🤦🏻‍♂ <- you already defined 5 as 4 and therefore do not need the const const in front
```
